src/coreclr/src/jit/importer.cpp

```diff
@@ -7526,11 +7526,13 @@ enum
     PREFIX_TAILCALL_EXPLICIT = 0x00000001, // call has "tail" IL prefix
     PREFIX_TAILCALL_IMPLICIT =
         0x00000010, // call is treated as having "tail" prefix even though there is no "tail" IL prefix
-    PREFIX_TAILCALL    = (PREFIX_TAILCALL_EXPLICIT | PREFIX_TAILCALL_IMPLICIT),
-    PREFIX_VOLATILE    = 0x00000100,
-    PREFIX_UNALIGNED   = 0x00001000,
-    PREFIX_CONSTRAINED = 0x00010000,
-    PREFIX_READONLY    = 0x00100000
+    PREFIX_TAILCALL_STRESS =
+        0x00000100, // call doesn't "tail" IL prefix but is treated as explicit because of tail call stress
+    PREFIX_TAILCALL    = (PREFIX_TAILCALL_EXPLICIT | PREFIX_TAILCALL_IMPLICIT | PREFIX_TAILCALL_STRESS),
+    PREFIX_VOLATILE    = 0x00001000,
+    PREFIX_UNALIGNED   = 0x00010000,
+    PREFIX_CONSTRAINED = 0x00100000,
+    PREFIX_READONLY    = 0x01000000
```

Comment:
Why do these flags only use every forth bit?
i.e. 0x1000, 0x2000, 0x4000, 0x8000

Reply:
I don't see any reason. I was just following the existing pattern.